test/pipelines/test_eval.py

```diff
@@ -200,7 +200,8 @@ def test_eval_pipeline(document_store, reader, retriever, samples_path):
     assert metrics_sas_cross_encoder["Reader"]["sas"] == pytest.approx(0.71063, 1e-4)
 
 
-@pytest.mark.parametrize("document_store", ["elasticsearch", "faiss", "memory"], indirect=True)
+@pytest.mark.unit
+@pytest.mark.parametrize("document_store", ["memory"], indirect=True)
 def test_eval_data_split_word(document_store, samples_path):
```

Comment:
This (and the following one) is not really a unit test, as you're creating a whole preprocessor and using it into `add_eval_data`. A unit test would just make sure that a preprocessor mocks gets used by `add_eval_data` in the expected way.

This is all to say that this one should be marked integration :slightly_smiling_face:

Reply:
@ZanSara I agree it's not a unit test in it's current form. However, I'd also say the test is not testing the integration of Haystack with 3rd party software. It's just testing Haystack. That's why I am against making it an integration test of pipelines in a rush. These tests were deactivated for a reason and the only integration tests we currently have are for PromptNode, Agent and the docstores here: https://github.com/deepset-ai/haystack/blob/4b380d8fb07a945c1aa45650b86e561c6790ccfc/.github/workflows/tests.yml#L764
Rewriting the tests as proper unit tests seems to be the best option here for most tests, no?